.github/workflows/staging.yml

```diff
@@ -1,12 +1,52 @@
-name: Staging Deployment
+name: Deploy Staging
 
-on: workflow_dispatch
+on:
+  workflow_dispatch:
+    inputs:
+      deployMain:
+        description: "Deploy main? true/false"
```

Comment:
@Ninamma Unsure if you were planning on making further updates to this but I was thinking the description for this could be something along the lines of:

`Deploy dev onto staging? (true/false)`